src/buffer/cylinder-buffer.ts

```diff
@@ -49,6 +49,12 @@ class CylinderBuffer {
   }
 }
 
+const CylinderBuffer: {
+  new(data: CylinderBufferData, params: Partial<CylinderBufferParameters>): CylinderGeometryBuffer | CylinderImpostorBuffer;
+} = CylinderBufferImpl as any;
```

Comment:
Instead of using the `any` type assertion, I would suggest to use a `//@ts-expect-error` comment. For example `//@ts-expect-error Incompatible parameters types between constructors` before the declaration to document the issue and maybe have it resolved through future evolutions of TS.

Reply:
Also, the informations provided by the JS-doc comment are now disconnected from CylinderBuffer. Please, move them before the `const` declaration.

Reply:
Sure, makes sense